package.json

```diff
@@ -17,19 +17,15 @@
         "docs",
         "python/perspective"
     ],
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/finos/perspective"
-    },
```

Comment:
Something weird about this diff since this is [not where this property is within the current `package.json`](https://github.com/finos/perspective/blob/86cd2c1f9135db586ed0a8f74bea1c7f50759dde/package.json#L3-L6).